packages/runtime/container-runtime/src/summary/orderedClientElection.ts

```diff
@@ -416,29 +416,26 @@ export class OrderedClientElection
 			change = true;
 		}
 		if (change) {
-			if (isSummarizerClient) {
-				this.logger.sendTelemetryEvent({
-					eventName: "SummarizerClientElected",
-					electedClientId: this._electedClient?.clientId,
-					electedParentId: this._electedParent?.clientId,
-					electionSequenceNumber: sequenceNumber,
-				});
-			}
+			this.logger.sendTelemetryEvent({
+				eventName: "SummarizerClientElected",
```

Comment:
Should we change the event name as it can be non summarizer client also?

Reply:
we could but I don't think we need it as it is just an easier way to identify them.